src/scripts/main.js

```diff
@@ -1,7 +1,13 @@
 'use strict';
 
 const pushNotification = (posTop, posRight, title, description, type) => {
-  // write code here
+  let element;
+
+  setTimeout(() => {
+    element = document.createElement('div');
```

Comment:
The `element` variable is declared and assigned a new `div` element, but it is not used further in the function. Consider appending this element to the DOM or using it according to your notification logic.